utils/git_export_campaign

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+"""A command similar to git-subtree, for exporting the history of a campaign.
+
+Hardcoded for the export of An Orcish Incursion, but added to the utils/ dir
+as other campaigns will likely be removed before 1.16. The result is a Git
```

Comment:
I'd prefer something less presumptive.

Reply:
realized that my above comment might not have been clear enough, it's referring to "...as other campaigns will likely be removed before 1.16. " something like "in case other campaigns are removed." seems better.